cmake/SociConfig.cmake

```diff
@@ -69,7 +69,7 @@ else()
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
         elseif (CMAKE_SYSTEM_NAME MATCHES "Windows")
             # We need to disable the warnings for using %I64d with MinGW.
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format")
+            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format -Wno-format")
```

Comment:
Sorry, I'm continuing to be lost (see previous comment)... What check are we speaking about here?

What puzzles me is why `-Wno-pedantic-ms-format` doesn't work for @snikulov: here it definitely does, i.e. with it the warning I see with just `-pedantic` is not given any more when using MinGW-w64 4.9.1.